test/system/casebooks_test.rb

```diff
@@ -26,53 +26,48 @@ class CasebookSystemTest < ApplicationSystemTestCase
       sign_in @user = users(:verified_professor)
     end
 
-    scenario 'creating a casebook', solr: true do
-      skip 
+    scenario 'creating a casebook', solr: true, js: true do
       visit root_path
 
-      ### clicking create casebook opens a modal instad of page navigation
-      find('.create-casebook').click
+      find('.create-casebook').trigger 'click'
       click_link 'From Scratch'
 
-      fill_in 'Title', with: 'Test casebook title'
-      fill_in 'Subtitle', with: 'Test casebook subtitle'
-      fill_in 'Headnote', with: 'Test casebook headnote'
+      fill_in 'content_casebook_title', with: 'Test casebook title'
+      fill_in 'content_casebook_subtitle', with: 'Test casebook subtitle'
+      fill_in 'content_casebook_headnote', with: 'Test casebook headnote'
 
-      click_button 'Save'
+      click_link 'Save'
```

Comment:
Pro tip for the future: You can use `click_on` to click a link or a button

Reply:
Nice